image_explorer/public/js/image_explorer.js

```diff
@@ -44,15 +44,20 @@ function ImageExplorerBlock(runtime, element) {
 
       if (active_feedback) {
         close_feedback();
+          // revealContainer.css('display' , 'block');
```

Comment:
Are there plans to use this line?
This will likely throw a `ReferenceError`as `revealContainer` isn't defined until line 52.